snuba/utils/metrics/backends/abstract.py

```diff
@@ -13,16 +13,26 @@ class MetricsBackend(ABC):
     def increment(
         self, name: str, value: Union[int, float] = 1, tags: Optional[Tags] = None
     ) -> None:
+        """
+        Emit a single instance of a non-timing metric that can be counted,
+        sliced/diced into percentiles, or summed
+        """
```

Comment:
This is a real nitpick but I think it's clearer as:
Increments a count metric by the `value` provided. (or something along those lines)

TIL we don't expose `decrement` function in here but otherwise your definition would be the same.

Reply:
Yeah, I guess I find the wording around metrics to be clumsy and not super-standardized though it's possible I'm not familiar with the standards.

If I'm in Datadog I'm often much more interested to view the p99 of a metric emitted like this vs. the sum so I don't really think of it as a counter (even if that's their term for it).

Reply:
I see, I believe this is often referred to as a "counter" metrics (at least in statsd). I think it may helpful to align the wording to that.

Reply:
I wonder if it would be helpful to add examples here. I'm fine with using stats/Datadog terminology but I'm thinking that coming in as a new person I didn't understand what you meant.

Reply:
Fair enough. Examples never hurt, up to you whether to add